README.md

```diff
@@ -1 +1,57 @@
-# ci-script
\ No newline at end of file
+# ci-script
+
+Script to help setup a repo with continuous integration support.
+
+- Uses GitHub Actions.
+- Actions configs for different setups.
+
+## Usage
+
+From your source dir, run:
+
+    ../ci-script/setup-ci MODE
+
+Where MODE is one of:
+
+- bedrock
+- bedrock-web
+- isomorphic
+- lint
+
+## Lint
+
+If you are using lint support:
+
+    npm i -D eslint
+    npm i -D eslint-config-digitalbazaar
+    # optional: eslint-plugin-jsdoc
+    # optional: eslint-plugin-vue
+    # setup a .eslintrc.js
+
+Add a `"script"` target something like:
```

Comment:
something like ...
Add a `lint` script target like: 

To ensure that people understand that the target MUST be called `lint`.